healthyregions/chicago-environment-explorer
# feat: added new page template for Learn pages (+ Instituto + PRN hidden pages)

## Problem
We would like to add new pages to ChiVes called "Learn", "Instituto", and "People's Response Network"

Details about these new pages are emerging, but for now we will need templates to begin modifying

## Approach
* feat: added a new top-level page to the Routes/Navbar - "Learn"
    * Top-level page contains an embedded tutorial video ([YouTube](https://www.youtube.com/watch?v=tSUBK-ZMXNs))
    * 3 sub-pages that include instructions, screenshots, and questions for participants to answer while working through the lesson
* feat: added 2 new sub-pages to the Routes/Navbar - "Instituto" / "People's Response Network"
    * ~~Currently nested under "My Community" (subject to change)~~
    * Currently hidden, pending additional content from our associates
* fix: update Shubham's job title to be more accurate
* fix: Various miscellaneous UI/UX updates
    * NavBar:
        * Red -> Teal, consistent for top-level links
        * Add margin-left / margin-right for sub-links
        * Color should be Green for active route
    * Learn / Explore:
        * Learn / Explore sub links on their own pages - header should be clickable
        * Consistent font size/color for headers (32px + teal)
    * Map:
        * Change the format for how the Overlays are displayed in the VariablePanel: " + A + B" -> "Overlay: A, B, C"
        * Add some margin/padding between Variables dropdown and "Overlays" text
        * "Overlay: " should appear in Pink
        * Overlay names in Grey / Black
        * Pink dropdown arrow -> Green (to denote clickable)
        * "Try with multiple variables" link should be teak + remove underline (consistent with link styling throughout the app)
        * fixed: text wrapping on VariablePanel
    * Contact:
        * Submit button should be primary brand color (fixes #109)
        * Adjust height of Message input (fixes #107)


## How to Test
1. Navigate to the homepage
    * You should see the new Home page content / layout has been drastically improved
2. Expand the Navbar dropdown at the top-left and click on "Learn"
    * You should be redirected to the new "Learn" page
    * You should see that the contents offer ChiVes-related tutorials
    * You should see a YouTube video embedded in this page
3. Click on Mapping 101 to view that tutorial
    * You should be brought to the Mapping 101 tutorial
    * You should see the tutorial includes instructions, screenshots, and questions for participants to answer
4. Go back to "Learn" and click on Histogram Filtering to view that tutorial
    * You should be brought to the Histogram Filtering tutorial
    * You should see the tutorial includes instructions, screenshots, and questions for participants to answer
5. Go back to "Learn" and click on Index Builder to view that tutorial
    * You should be brought to the Index Builder tutorial
    * You should see the tutorial includes instructions, screenshots, and questions for participants to answer
